packages/utils/src/libp2p/index.ts

```diff
@@ -20,3 +27,37 @@ export async function convertPubKeyFromPeerId(peerId: PeerId): Promise<PublicKey
 export async function convertPubKeyFromB58String(b58string: string): Promise<PublicKey> {
   return await convertPubKeyFromPeerId(PeerId.createFromB58String(b58string))
 }
+
+/**
+ * 
+ * Returns true or false if given string does not contain a b58string
+ * 
+ * @param string arbitrary content with maybe a b58string
+ */
+export function hasB58String(content: string): Boolean {
+  const hasMatcheableContent = content.match(b58StringRegex)
+  if(hasMatcheableContent) {
+    const [maybeB58String] = hasMatcheableContent
+    const b58String = maybeB58String.substr(0, 53)
+    return b58String.length === 53
+  } else {
+    return false
+  }
+}
+
+/**
+ * 
+ * Returns the b58String within a given content. Returns empty string if none is found.
+ * 
+ * @param string arbitrary content with maybe a b58string
+ */
+export function getB58String(content: string): string {
+  const hasMatcheableContent = content.match(b58StringRegex)
+  if(hasMatcheableContent) {
+    const [maybeB58String] = hasMatcheableContent
+    const b58String = maybeB58String.substr(0, 53)
+    return b58String
+  } else {
+    return ''
```

Comment:
not important, but is there a benefit of returning `''` instead `undefined` ?

Reply:
We can check if it's invalid by its `length` and keeping the same type.